pkg/apis/contrail/v1alpha1/templates/vrouter_config.go

```diff
@@ -55,13 +55,13 @@ var ContrailCNIConfig = template.Must(template.New("").Parse(`{
   "contrail" : {
```

Comment:
Can you move it out of the vrouter_config.go in to a separate cni_config file or something? If we separate the Custom Resources, we should separate the configs as well.

Reply:
Done